src/proxy/velocity.rs

```diff
@@ -0,0 +1,5 @@
+pub static VELOCITY_PLAYER_INFO_CHANNEL: &str = "velocity:player_info";
+pub const VELOCITY_SUPPORTED_VERSION: i32 = 1;
```

Comment:
Considering that Valence is a library, I'm a bit concerned about what the implications of this is for semver compatibility. Is updating support for velocity/bungeecord/waterfall a breaking change? Are versions of velocity/bungeecord/waterfall backwards compatible?